src/main/webapp/WEB-INF/pages/error.jsp

```diff
@@ -3,6 +3,6 @@
 <%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
 <%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
 
-<tags:master pageTitle="404 error">
-  <h1>404 Not Found Page </h1>
+<tags:master pageTitle="Error">
```

Comment:
why did you create one common page for 404 and 500 errors? These are different errors